src/common/components/InputField/Input/Input.tsx

```diff
@@ -0,0 +1,44 @@
+import { ChangeEvent } from "react";
+import classNames from "classnames/bind";
+import styles from "./Input.module.scss";
```

Comment:
import 개행도 신경써보시면 좋습니다~
찾아보시면 나름대로 구분짓는 룰도 있고 본인만의 규칙을 만들어보시면 좋습니다~
```suggestion
import { ChangeEvent } from "react";
import classNames from "classnames/bind";

import styles from "./Input.module.scss";
```